components/IngredientSelector/Navigator.tsx

```diff
@@ -130,9 +134,60 @@ export default function Navigator() {
           );
         })}
       </InteractionWrapper>
+    
     );
   }
-  return (
+
+  if (view === "value") {
```

Comment:
You don't need this `if` since it's the only remaining value after all the other `if`